Sources/Foundation/Process.swift

```diff
@@ -939,21 +939,17 @@ open class Process: NSObject, @unchecked Sendable {
             useFallbackChdir = false
         }
 
-#if canImport(Darwin) || os(Android) || os(OpenBSD) || os(FreeBSD)
-        var spawnAttrs: posix_spawnattr_t? = nil
-#else
-        var spawnAttrs: posix_spawnattr_t = posix_spawnattr_t()
-#endif
-        try _throwIfPosixError(posix_spawnattr_init(&spawnAttrs))
+        let spawnAttrs = _CFPosixSpawnAttrAlloc()
+        try _throwIfPosixError(_CFPosixSpawnAttrInit(spawnAttrs))
 #if os(Android)
```

Comment:
Remove this Android check below, it should no longer be needed.

@Steelskin, can you run this pull through the TBC CI, with this block removed, and let us know if it passes?